beacon-chain/forkchoice/doubly-linked-tree/store_test.go

```diff
@@ -82,7 +82,7 @@ func TestStore_Head_Itself(t *testing.T) {
 	require.NoError(t, err)
 	require.NoError(t, f.InsertNode(context.Background(), state, blkRoot))
 
-	// Since the justified node does not have a best descendant so the best node
+	// Since the justified node does not have the best descendant so the best node
```

Comment:
```suggestion
	// Since the justified node does not have a best descendant, the best node
```
I think `a` is correct here, similarly how you would say "I have a best friend", not "I have the best friend".

Reply:
`a` is correct here